### ISSUE TITLE
[Feature] example with Nuxt.js

### ISSUE THREAD

**Issue by frankspin89** [graphcool-examples/vue-graphql#19]
Would be awesome to see an example to use Graph.cool with Nuxt.js. 

I"m curious to see how to implement GraphQL with Server Side rendering

**Comment by dohomi** [comment]
Hi @frankspin89 there are couple of discussions about apollo and vue/nuxt. Here is one approach which works pretty well if you are using it with vuex: https://github.com/PierBover/vuex-apollo-example-project
Otherwise you would need to wait till nuxt 1.x is finalized with an working example of vue-apollo. You can find the discussion here:
https://github.com/nuxt/nuxt.js/issues/504
I am using the vanilla javascript approach of Pier and it works very neat. If you are using vuex in your app give it a shot

**Comment by frankspin89** [comment]
@dohomi thanks! I started to implement the approach of Pier. The basics are working fine for me. Had to disable subscriptions on the server side.

**Comment by dohomi** [comment]
@frankspin89 yes I am using subscriptions only on the client side: https://github.com/nuxt/nuxt.js/issues/984 I also couldn't require ws on the server side

**Comment by frankspin89** [comment]
@dohomi Do you also have to use fetchPolicy: 'network-only' to load new data correctly?

**Comment by dohomi** [comment]
@frankspin89 no that I only had to use on vue-apollo, but I removed that from my project. With vanilla apollo I do:
```
  const ssr = process.SERVER_BUILD
  return new ApolloClient({
    networkInterface: networkConf,
    dataIdFromObject: r => r.id,
    ...(ssr ? {
      ssrMode: true
    } : {
      connectToDevTools: process.env.NODE_ENV !== 'production'
      // ,
      // initialState
    })
  })
```

**Comment by vuchl** [comment]
thee is this https://github.com/nuxt/nuxt.js/pull/1160 and this https://github.com/nuxt/nuxt.js/pull/1161 now

**Comment by marktani** [comment]
That's awesome, thanks @vuchl!

**Comment by dohomi** [comment]
an official example came out now for rc2: https://github.com/nuxt/nuxt.js/tree/dev/examples/vue-apollo

**Comment by vuchl** [comment]
yes the newly release vue-apollo is much smarter: there is now a nuxt module (nuxt-community/apollo-module)[https://github.com/nuxt-community/apollo-module] that is responsible to provide the apollo client in the nuxt environment. and you only need to specify the graphql endpoint in your nuxt.config.js. Really well architected.

**Comment by frankspin89** [comment]
Maybe i'm missing something but the new release seems far from complete? Don't see a option to authenticate requests with a bearer token? And it's lacking a implementation to use subscriptions?

**Comment by vuchl** [comment]
it's just a simple example of how to integrate the vue-apollo plugin as a module in nuxt. Detailed Auth examples and mutations are not yet in.

**Comment by marktani** [comment]
Here's a great Nuxt example: https://www.graph.cool/forum/t/nuxt-js-apollo-website-for-sharing-codes/545?u=nilan :) Thanks to @Samuell1 for working on that! :tada:

**Comment by sebastianmacias** [comment]
Any update on this?

Tnx

**Comment by dohomi** [comment]
I'd recommend to use https://github.com/nuxt-community/apollo-module

**Comment by Lahori-Jawan** [comment]
@sebastianmacias @dohomi @frankspin89 Although pretty late but I had built a blog using node+nuxt+graphQL(apollo)+sequelize [here](https://github.com/Lahori-Jawan/graphX)
